### PR TITLE
run nightly nightly

### PR DESCRIPTION
**Pull request by mikemhenry**
Instead of including checks against nightly openmm builds on ever PR, we will run checks every night and if they fail make an issue on github

This PR will be smaller when PR #760  and PR #763 get merged in.

### CODE REVIEW

**Comment by mikemhenry** [comment]
@jaimergp Not exactly what is wrong with the syntax in the new Nightly.yml file, but my idea was to move the nightly testing as part of a cron job and not test PRs against nightly. This _should_ make a github issue when nightly fails, which should help keep those issues visible.

File: .github/workflows/Nightly.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          nosetests perses --with-coverage --cover-package=perses --verbosity=3 --with-timer -a '!advanced' --cover-xml --cover-xml-file=./coverage.xml
+          popd
+
+      - name: Make an issue
+      - uses: JasonEtco/create-an-issue@v2
+        if: ${{ failure() }}
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+             assignees: mikemhnery
```
Comment (jaimergp) [inline .github/workflows/Nightly.yml:99]:
```suggestion
             assignees: mikemhenry
```

:)

**Comment by jaimergp** [comment]
What if instead of having different workflows we use conditionals based on [`github.event_name`](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context)?

File: .github/workflows/Nightly.yml (hunk 1 above)
Comment (mikemhenry) [reply .github/workflows/Nightly.yml]: oops :name_badge:

**Comment by mikemhenry** [comment]
That could work and cuts down on the duplication. At one point I split the checks into a new workflow so they wouldn't block auto merging or make it look like our build was failing when it wasn't, but I think marking the checks as not required should fix that, I'll give it a try.

File: .github/workflows/Nightly.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+on:
+  schedule:
+    # nightly tests
+    - cron: "0 0 * * *"
+  # Be able to run manually
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Test on ${{ matrix.cfg.os }}, Python ${{ matrix.cfg.python-version }}, OpenMM ${{ matrix.cfg.openmm }}
```
Comment (jaimergp) [inline .github/workflows/Nightly.yml:12]: What about this, but on `.github/workflows/CI.yaml`?

```suggestion
    if: (github.event_name == 'schedule'  && matrix.cfg.nightly) || github.event_name != 'schedule' 
    name: Test on ${{ matrix.cfg.os }}, Python ${{ matrix.cfg.python-version }}, OpenMM ${{ matrix.cfg.openmm }}
```

You'll need a `nightly` key on the relevant matrix entries.

Comment (jaimergp) [reply .github/workflows/Nightly.yml:12]: The individual steps will need to check against `if: matrix.cfg.nightly` only.

Comment (mikemhenry) [reply .github/workflows/Nightly.yml:12]: I cut those when at first I was dropping nightly from running on PRs

**Comment by mikemhenry** [comment]
Part of what I'm trying to do is not have our build badge be red, but I think I've fixed that by chancing which checks are required

**Comment by mikemhenry** [comment]
Going to work on making nightly work

**Pull request closed without merging**